tests/assembly/static-relocation-model.rs

```diff
@@ -25,8 +25,7 @@ extern "C" {
 }
 
 // CHECK-LABEL: banana:
-// On the next line LLVM 14 produces a `movb`, whereas LLVM 15+ produces a `movzbl`.
-// x64: {{movb|movzbl}}   chaenomeles{{(\(%[a-z0-9]+\))?}}, %{{[a-z0-9]+}}
+// x64:      movzbl  chaenomeles{{(\(%[a-z0-9]+\))?}}, %{{[a-z0-9]+}}
```

Comment:
I removed this flexible match because it spoke of very old versions, but it seems the `nopt` CI reaches this "old" output too. I'll restore that and make the comment more general about how it may vary.